examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>May you also explain when it might be useful to use this method(series in the bottom level are noise) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>